core/services/ardupilot_manager/settings.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pathlib
+
+import appdirs
+
+
+class Settings:
+    app_name = "ardupilot-manager"
+
+    def __init__(self) -> None:
+        # Create folder for ardupilot-manager service if it doesn't exist
+        self.settings_path = appdirs.user_config_dir(Settings.app_name)
+        os.makedirs(self.settings_path, exist_ok=True)
```

Comment:
missing try catch

Reply:
In the case we cannot make a settings directory, do you think it would be better to say to the user he will have limited operation or raise the error and stop the service?

Reply:
I think that is better to raise the error and continue the operation, with that will at least will have the log of what happen and the user will still be able to use the ROV

Reply:
Done